meshtastic/mesh_interface_runtime/receive_pipeline.py

```
@@ -0,0 +1,776 @@
+"""Receive pipeline for processing inbound packets from the radio."""
+
+import base64
+import copy
+import hashlib
+import logging
+import threading
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING, Any, Callable, TypeAlias
```

Ruff (0.15.7)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)